graphql_compiler/query_formatting/match_formatting.py

```diff
@@ -50,6 +51,20 @@ def _safe_match_date_and_datetime(graphql_type, expected_python_types, value):
     return _safe_match_string(serialized_value)
 
 
+def _safe_match_decimal(value):
+    """Represent decimal objects as MATCH strings."""
+    if isinstance(value, Decimal):
```

Comment:
this code has been repeated twice, ideally there would be a function 

```
def coerce_decimal(value):
  if isinstance(value, Decimal):
     return value
  else:
    try:
￼       return Decimal(value)
       except InvalidOperation as e:
￼     raise GraphQLInvalidArgumentError(e)
```
that gets called from both gremlin and match generator